sdk/eventhubs/ci.yml

```diff
@@ -48,3 +48,9 @@ extends:
       - name: azure-messaging-eventhubs-checkpointstore-blob
         groupId: com.azure
         safeName: azuremessagingeventhubscheckpointstoreblob
+      - name: azure-messaging-eventhubs-track1-perf
```

Comment:
You'll want to add these under `AdditionalModules`, similar to this: https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/storage/ci.yml#L81

May also need to add these as additional modules to tests.yml